index.html

```diff
@@ -556,6 +556,30 @@ <h3 id="navigation-scope-security-considerations">
       </p>
       <section>
         <h3>
+          Privacy and security considerations
+        </h3>
+        <p>
+          When the <a>fullscreen</a> <a>display mode</a> is applied, it is
+          RECOMMENDED that the user agent still afford the end-user a means to
+          access/view the origin, or URL, and icon of the Web application. This
```

Comment:
... RECOMMENDED that the user agent provides the end-user a means to consult basic information about the web apps, such as the origin, complete URL, granted permissions and associated icon.

Such information shouldn't necessarily need to overlay the application itself, but can instead be shown in the task switcher, for instance by clicking on an info button.